static/js/modules/global-nav.js

```diff
@@ -0,0 +1,2 @@
websites</h5>\n        <ul class="global-nav__split-list">\n          '+l.map(function(a){return i(a)}).join("")+'\n        </ul>\n      </div>\n      <div class="global-nav__row">\n        <h5 class="global-nav__muted-heading global-nav__expanding-row">Resources</h5>\n        <ul class="global-nav__split-list">\n          '+o.map(function(a){return i(a)}).join("")+'\n        </ul>\n      </div>\n      <div class="global-nav__row">\n        <h5 class="global-nav__muted-heading global-nav__expanding-row">About</h5>\n        <ul class="global-nav__split-list u-no-margin--bottom">\n          '+t.map(function(a){return i(a)}).join("")+"\n        </ul>\n      </div>\n    </div>"}(d)+'\n    <div class="global-nav__strip u-hide--mobile">\n      <div class="global-nav__row is-bordered">\n        <ul class="global-nav__matrix">\n          '+m+'\n        </ul>\n      </div>\n    </div>\n    <div class="global-nav__strip u-hide--mobile">\n      <div class="global-nav__row">\n        <div class="global-nav__flex-container">\n          <div class="global-nav__others-col">\n            <h5 class="global-nav__muted-heading">Other websites</h5>\n            <div class="global-nav__matrix">\n              '+u+'\n            </div>\n          </div>\n          <div class="global-nav__resources-col">\n            <h5 class="global-nav__muted-heading">Resources</h5>\n            <ul class="global-nav__split-list">\n              '+c+'\n            </ul>\n            <h5 class="global-nav__muted-heading">About</h5>\n            <ul class="global-nav__split-list">\n              '+_+"\n            </ul>\n          </div>\n        </div>\n      </div>\n    </div>")+"\n      </div>\n      "+s+"\n    </div>");document.body.insertBefore(e,document.body.firstElementChild),e.appendChild(f),e.appendChild(w),e.appendChild(r),function(a,o){var l=o.querySelectorAll(".global-nav__header-link"),t=o.querySelector(".global-nav__dropdown"),i=o.querySelectorAll(".global-nav__dropdown-content"),n=o.querySelectorAll(".global-nav__expanding-row"),e=o.querySelector(".global-nav__overlay"),r=window.innerWidth<a;function s(){t.classList.remove("show-content"),l.forEach(function(a){return a.classList.remove("is-selected")}),e.classList.remove("show-overlay")}function g(){window.scrollTo(0,o.offsetTop)}function d(a){var n=a.querySelector(".global-nav__header-link-anchor").getAttribute("href"),l=o.querySelector(n);a.classList.add("is-selected"),i.forEach(function(a){return a!==l&&a.classList.add("u-hide")}),l.classList.remove("u-hide"),e.classList.add("show-overlay"),r&&g()}l.forEach(function(n){n.addEventListener("click",function(a){a.preventDefault(),t.classList.contains("show-content")?n.classList.contains("is-selected")?s():(l.forEach(function(a){return a.classList.remove("is-selected")}),d(n)):(t.classList.add("show-content"),d(n))})}),n.forEach(function(a){a.addEventListener("click",function(a){a.target.classList.toggle("is-active"),g()})}),e.addEventListener("click",s)}(900,e)},a}({});
```

Comment:
This should also be `git ignore`d. Again, not sure what happened, why it's not already.